README.md

```diff
@@ -137,11 +140,36 @@ The token proxy is what is called by the SMART client to exchange it's authoriza
 
 11.  App calls the /token proxy to obtain an access token.​
 
-12.  /token proxy obtains the token from Okta, uses a private key to perform client authentication (public apps), or by forwarding client id/secret (confidential apps)
+12.  /token proxy obtains the token from Okta, using a private key to perform client authentication (public apps), or by forwarding client id/secret (confidential apps).
+
+13.  If Okta returns a refresh token to the /token proxy, the id of the refresh token, as well as any selected patient id are then stored in the launch response cache to be replayed again at refresh time.
+
+14.  /token proxy grabs the selected patient id out of the access token, and decorates it's response JSON with the patient id if requested.
+
+15.  App accesses the FHIR API.
+
+## Refresh Flow
+![Refresh Flow](./images/SMART_FHIR_Reference_Refresh_Flow.png "Refresh Flow")
+
+1.  User accesses a SMART-enabled app. The application presents the user with a secure way of accessing the refresh token from device memory (biometric or similar).
+
+2.  The app invokes the /token proxy endpoint passing along the refresh token as well as client authentication (client_id/client_secret).
+
+3. The /token proxy passes the request along to Okta to request a new id/access token.
+
+4. Okta invokes the token hook.
+
+5. The token hook uses the refresh token id to retrieve any runtime cached information, such as patient id or other launch response information. This information is provided to Okta to include in the newly refreshed access token.
+
+6. Okta mints a new access token, including the retrieved cached information if any.
+
+7. The /token proxy will again re-insert the refreshed data back into the launch response cache. This achieves 2 things:
+* It will update the TTL on the record to indicate activity on the record so it will not expire.
+* Depending upon Okta configuration, Okta may issue a brand new refresh token upon every refresh request (versus reusing the original one). If this occurs, a new record will be required to be inserted into the cache. This logic is the same logic from step 13 in the authorization flow above.
```

Comment:
I think if you add two spaces (as I did below) the bullets will be indented under number 7 (looks a bit nicer)
```suggestion
  * It will update the TTL on the record to indicate activity on the record so it will not expire.
  * Depending upon Okta configuration, Okta may issue a brand new refresh token upon every refresh request (versus reusing the original one). If this occurs, a new record will be required to be inserted into the cache. This logic is the same logic from step 13 in the authorization flow above.
```

Reply:
Thanks! I've made this change (had to add 4 spaces due to it being in a list).